src/Host/Broker/Impl/Startup/Startup.cs

```diff
@@ -62,7 +64,13 @@ SecurityManager securityManager
             });
 
             app.Use((context, next) => {
-                if (!context.User.Identity.IsAuthenticated) {
+                if (context.Request.Path.Value.Contains("/remoteuri")) {
```

Comment:
The only thing it needs is routing, which is available separately of full-fledged controllers. But it should use `MapRoute`.

https://docs.asp.net/en/latest/fundamentals/routing.html#using-routing-middleware